packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -201,17 +201,13 @@ class PerformanceController extends DisposableController
       data?.displayRefreshRate = _displayRefreshRate.value;
 
       // Listen for Flutter.Frame events with frame timing data.
+      // Listen for Flutter.RebuiltWidgets events.
       autoDispose(
           serviceManager.service.onExtensionEventWithHistory.listen((event) {
         if (event.extensionKind == 'Flutter.Frame') {
           final frame = FlutterFrame.parse(event.extensionData.data);
           addFrame(frame);
-        }
-      }));
-
-      // Listen for Flutter.RebuiltWidgets events.
-      autoDispose(serviceManager.service.onExtensionEvent.listen((event) {
-        if (event.extensionKind == 'Flutter.RebuiltWidgets') {
+        } else if (event.extensionKind == 'Flutter.RebuiltWidgets') {
```

Comment:
not a request but just a comment to think about for the future - it would be great if there is a way to connect rebuild stats with the flutter frame they occurred in.

Reply:
For sure. We have a 'startTime' info field for the rebuild events, but I don't know if that's reliable enough for us to use in terms of matching the events up with a frame.

We could also change the rebuild event to include a frame number, which would make things much easier for downstream tools. I assume the number would be available to the place where we generate rebuild events, though it might take a bit of framework surgery to expose it.